service/cloudFormation/sam-template.yml

```diff
@@ -44,7 +44,7 @@ Resources:
       Description: Lambda Function to run playwright test
       FunctionName: !Ref LambdaTestName
       Role: !GetAtt syntheticQueryS3Bucket.Arn
-      MemorySize: 512
+      MemorySize: 1024
```

Comment:
Do you think maybe we should make it configurable from the user's side, with a default value of 1024? The same goes for the timeout parameter.